meteor/override-fastclick.js

```diff
@@ -0,0 +1,6 @@
+// disable fastclick package in Meteor if included
```

Comment:
We can also rename to folder to something more general like `packaging` or place it inside `scripts` or `config` which has build specific things